woocommerce/woocommerce-android
# Add toolbar to edit shipping payment screen

Part of #10570 

### Description

This PR introduces individual toolbars for edit shipping payment screen, replacing the shared toolbar previously used with MainActivity. This modification is essential to accommodate screens within a two-pane layout, ensuring that each screen maintains its distinct toolbar despite sharing a common parent.

### Testing instructions

Navigate to the Order detail screen of an Order that has a shipping label (PbTz5e-11v-p2) (Order list -> Order detail -> Create a shipping label CTA -> Edit payment details) screen and ensure the toolbar and its action work as expected on both phone and tablet.

### Images/gif

<img width="244" alt="Screenshot 2024-02-12 at 7 06 42 PM" src="https://github.com/woocommerce/woocommerce-android/assets/1331230/605fba60-9f12-40d3-90ce-e3b84f759c00">
